syntax = "proto3";

package eds;

import "google/protobuf/wrappers.proto";

service EdsService {
  rpc ExecuteSignByEds (SignByEdsStart) returns (SignByEdsResult);
  rpc SendStatus (EdsManagerStatusCheck) returns (stream EdsManagerStatus);
}

message SignByEdsStart {
  string type = 1;
}

message SignByEdsResult {
  string message = 1;
}

message EdsManagerStatusCheck {}

message EdsManagerStatus {
  google.protobuf.BoolValue busy = 1;
}
